CONTRIBUTING.md

```diff
@@ -230,6 +215,4 @@ or
 npm run make:deploy
 ```
 
-### Pull Request Images
-
-All pull requests will have an associated `pr-<PULL REQUEST NUMBER>` image built and pushed to [quay.io](https://quay.io/repository/opendatahub/odh-dashboard) for use in testing and verifying code changes as part of the PR code review.  Any updates to the PR code will automatically trigger a new PR image build, replacing the previous hash that was referenced by `pr-<PULL REQUEST NUMBER>`.
+you will deploy all the resources located in the `manifest` folder alongside the image you selected in the previous step.
```

Comment:
Minor typo:

`manifests` folder 

Sorry I missed this the first time.

Reply:
thanks a lot!